localization/english/wc_custom_titles_l_english.yml

```diff
@@ -10056,8 +10056,8 @@
  d_northcape_adj:0 "Northcapean"
  d_northfold:0 "Northfold"
  d_northfold_adj:0 "Northfoldian"
- d_northwoods:0 "Northwoods"
- d_northwoods_adj:0 "Northwoodsian"
+ d_northwoods:0 "Northshire"
+ d_northwoods_adj:0 "Northshire"
```

Comment:
according to http://www.peoplefrom.co.uk : 
```
BERKSHIRE - Berkshiremen, Berkshirewomen
BUCKINGHAMSHIRE - Buckinghamshiremen, Buckinghamshirewomen
BEDFORDSHIRE - Bedfordshiremen, Bedfordshirewomen
DERBYSHIRE - Derbyshiremen and Derbyshirewomen
HAMPSHIRE - Hampshiremen and Hampshire women
YORKSHIRE - Yorkshiremen and Yorkshirewomen
``` 
alternatively: 
```
Aberdeenshire | Aberdeen | Aberdonians
Lancashire | Lancaster | Lancastrians
```

therefore I suggest: `Northshireman` for the adjective.

Reply:
pushed